include/picongpu/fields/MaxwellSolver/Yee/Curl.hpp

```diff
@@ -35,9 +35,10 @@ namespace maxwellSolver
 namespace yee
 {
 
-    template< typename  Difference >
+    template< typename T_Difference >
```

Comment:
Even if this change is small please provide this change as a separate PR because it is independent to this PR.